src/main/java/core/basesyntax/service/ShopServiceImpl.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.service;
+
+import core.basesyntax.operation.OperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.transaction.FruitTransaction;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ShopServiceImpl implements ShopService {

```

Comment:
Let’s follow the single responsibility principle. Create a separate class that will be responsible for storing the fruit and provides methods to access it. In this class, let's leave the transaction processing logic (process method)